src/main/java/org/spongepowered/common/mixin/core/entity/EntityMixin.java

```diff
@@ -754,12 +741,12 @@ public ItemEntity entityDropItem(final ItemStack stack, final float offsetY) {
     @Nullable
     @Override
     public BlockPos bridge$getLastCollidedBlockPos() {
-        return this.lastCollidedBlockPos;
+        return this.impl$lastCollidedBlockPos;
     }
 
     @Override
     public Timing bridge$getTimingsHandler() {
-        return this.entityType.getTimingsHandler();
+        return ((EntityTypeBridge) this.shadow$getType()).bridge$getTimings();
```

Comment:
Is it on `EntityTypeBridge`? Did you add that or is that being inherited from `TimingsHolder` (or whatever the interface is)?

Reply:
There isn't a TimingsHolder interface but there is a timings instance inside of EntityTypeMixin which is what this bridge refers to.

Reply:
I see, I stand corrected. I was vaguely remembering timings to be instanced based on the bridge interface separate from the targeted bridges, but that's a separate story